### PR TITLE
update am pm display and titles

### PR DESCRIPTION
**Pull request by kanon-inoue**
changed font size of titles 
changed the am and pm as uppercase

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "configurations": [
+    {
+      "type": "chrome",
+      "request": "launch",
+      "name": "Launch Chrome against localhost",
+      "url": "http://localhost:8080",
+      "webRoot": "${workspaceFolder}"
+    }
+  ]
+}
```
Comment (HaruKawamata) [inline .vscode/launch.json]: どうやってこのファイルを作ったかわかる？できれば削除した方がいいと思う（謎の自動的に作られたファイルを追加しないほうがいい）

**Review by HaruKawamata — COMMENTED** [review]
ほとんどすごい！一個の質問があった

File: .vscode/launch.json (hunk 1 above)
Comment (kanon-inoue) [reply .vscode/launch.json]: i didnt add it on purpose. when i run any code, it appears in my code every time.

Comment (kanon-inoue) [reply .vscode/launch.json]: you can delete it

Comment (HaruKawamata) [reply .vscode/launch.json]: Got it! There's a way you can prevent it from being added to your commits - update the `.gitignore` file and add this line
```
.vscode/*
```

**Comment by kanon-inoue** [comment]
should i delete the .vscode file?

**Review by HaruKawamata — APPROVED** [review]
Nice work!

**Pull request merged**